packages/ListBox/src/components/Option/Option.styles.js

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+
+const blueSelected = "#e5f1fe";
+
+export const OptionStyled = styled.li`
+  padding: 4px;
+  margin-bottom: 4px;
+  font-size: 13px;
+  border-radius: 3px;
+
+  &:hover {
+    ${props => {
+      const isSelected = props.isSelected ? `background: ${blueSelected}` : `background: #f0f0f0`;
+      const isDisabled = props.isDisabled ? "background: transparent; cursor: not-allowed; border: 0;" : "";
+      const hasPreventDefaultOnSelect = props.hasPreventDefaultOnSelect ? "background: transparent;" : "";
+
+      return `
+        ${isSelected}
+        ${isDisabled}
+        ${hasPreventDefaultOnSelect}
+      `;
+    }}
+  }
+
+  ${props => {
+    const isDisabled = props.isDisabled
+      ? `color: ${tokens.color.blackLighten60}; border: 0; background: transparent;`
+      : "";
+
+    const isActived = props.isActive
+      ? `
+        border: 3px solid Highlight;
```

Comment:
Highlighted option content is displaying content in a slightly different position to non highlighted options content. Suggest using `outline` instead of `border` to prevent this affect the layout
![image](https://user-images.githubusercontent.com/4040102/59455302-b9b2f700-8dc8-11e9-9743-7e58febfb0e0.png)

Reply:
I don't think this is the right focus style at all, but this PR is not so concerned with the styling.  There will be a follow up PR to get it to ✨